<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VCDViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Open VCD File</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lblFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>571</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFileDialog">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>88</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>VCD Timescale:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lblTimescale">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>VCD Signals</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>29</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListView" name="vcdSignalNameListView"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>plot field</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>29</y>
       <width>221</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="plotSignalListView"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAddSignal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Signal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddPwl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add PWL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>470</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
